scrapbook-gateway-service/app.py

```diff
@@ -6,20 +6,50 @@
 
 from routes.auth import authenticate_user_api
 from routes.session import logout_user_api
-from routes.album import album_user_api
+from routes.album import album_create_api, album_list_api, album_fetchall_album_api, album_retrieve_images_api
+from routes.image import image_upload_api, image_download_api
 
 
 # Defines the Flask app
 app = Flask(__name__)
 # Cross Origin Resource Sharing (CORS) is enabled for the app
 CORS(app)
 
+#retries user info from the user service, and adds the use to a new session.
 app.register_blueprint(authenticate_user_api)
 
+#removes user from session
 app.register_blueprint(logout_user_api)
```

Comment:
Don't make a blueprint fr this. Logout should be in the AUTHENTICATE_USER_API, because logout is authentication function